src/Tizen.NUI/src/devel/Binding/ViewBindings.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Binding
+{
+    /// <summary>
+    /// Provides a set of static properties that represent the binding properties of the <see cref="View"/> class.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class ViewBindings
+    {
+        /// <summary>
+        /// Gets the binding property for the width of a <see cref="View"/>.
+        /// </summary>
+        public static BindingProperty<View, float> WidthProperty { get; } = new BindingProperty<View, float>
+        {
+            Setter = (v, value) => v.SizeWidth = value,
```

Comment:
I have a query. will all public properties in View be added into ViewBindings like WidthProperty here?

Reply:
it seems that all public properties which have type BindableProperty in View need be added into ViewBindings.

Reply:
Another query, when is only Setter (OneWay) added? when are both Setter & Getter (TwoWay) are added?
How to distinguish them?

Reply:
1. ViewBindings should have essential property only. as application can create their own BindingProperty in their code, we do not need to provide all property here.
2. TwoWay is only possible when the property notify their changes. TextField, TextEditor notify the Text property change in the TextChanged Event, so it is provided TwoWay, but Text property in TextLabel won't send any event so we can only provide OneWay. most cases, two way is not necessary and the case that real user can modify the property value (similar like editor... ) Two way would be necessary.

Reply:
got it. thank you for your detailed explanation.